### DISCUSSION TITLE
Disabling bracket pair highlight timer on bracket close?

### DISCUSSION THREAD

**Discussion by starptr** [nix-community/kickstart-nix.nvim#42] [Q&A] [answered]
When I type `}`, the editor highlights the paired `{` for a few hundred milliseconds, during which the cursor is invisible. This is only a cosmetic feature; I am able to type as usual during those hundreds of milliseconds but I would like to disable it. What feature in the base kickstart config is this coming from? Thanks!

https://github.com/user-attachments/assets/f19b4fb1-715c-4d28-912a-6630b3dae121

**Comment by mrcjkb** [comment]
Hey :wave:

That is a built-in Neovim feature.
See [`:h matchparen`](https://neovim.io/doc/user/pi_paren.html).

**Reply by starptr** [reply]
Hm I should clarify, I don't mean the light highlighting of both brackets, I meant the short moment where the opening bracket is brightly highlighted:

![CleanShot 2024-09-20 at 01 54 42@2x](https://github.com/user-attachments/assets/f001db96-d65a-4481-9724-5c825ec8ee3d)

Correct me if I'm wrong, but I thought the built-in Pi_paren plugin is only responsible for the light highlighting:

![CleanShot 2024-09-20 at 01 56 10@2x](https://github.com/user-attachments/assets/a1e1b850-9eea-475a-9646-86d787458c7f)

I'm only trying to disable the first behavior where the first bracket is brightly highlighted. I couldn't find explanation about it in the matchparen docs :(

**Reply by mrcjkb** [reply, marked as answer]
Right. It's the `vim.opt.showmatch` option, which is set in the `init.lua`.

**Reply by starptr** [reply]
That worked, thanks!
